vars/protecodeExecuteScan.groovy

```diff
@@ -0,0 +1,135 @@
+import com.sap.piper.GenerateDocumentation
+import com.sap.piper.JenkinsUtils
+import com.sap.piper.PiperGoUtils
+import com.sap.piper.Utils
+
+import groovy.transform.Field
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String STEP_NAME = getClass().getName()
+@Field String METADATA_FILE = 'metadata/protecode.yaml'
+@Field Set GENERAL_CONFIG_KEYS = [
+    /**
+     * Whether to create a side bar link pointing to the report produced by Protecode or not
+     * @possibleValues `true`, `false`
+     **/
+    'addSideBarLink',
+    /**
+     * Decides which parts are removed from the Protecode backend after the scan
+     **/
+    'cleanupMode',
+    /** The ID to the credential used for downloading the docker image from artifactory to scan it with Protecode */
+    'dockerCredentialsId',
+    /** The reference to the docker image to scan with Protecode */
+    'dockerImage',
+    /** The reference to the docker registry to scan with Protecode */
+    'dockerRegistryUrl',
+    /** The URL to fetch the file to scan with Protecode which must be accessible via public HTTP GET request */
+    'fetchUrl',
+    /** The path to the file from local workspace to scan with Protecode */
+    'filePath',
+    /** The URL to the Protecode backend */
+    'protecodeServerUrl',
+    /** The ID of the credentials used to access the Protecode backend */
+    'protecodeCredentialsId',
+    /** DEPRECATED: Do use triaging within the Protecode UI instead */
+    'protecodeExcludeCVEs',
+    /**
+     * Whether to fail the job on severe vulnerabilties or not
+     * @possibleValues `true`, `false`
+     **/
+    'protecodeFailOnSevereVulnerabilities',
+    /** The Protecode group ID of your team */
+    'protecodeGroup',
+    /** The timeout to wait for the scan to finish */
+    'protecodeTimeoutMinutes',
+    /** The file name of the report to be created */
+    'reportFileName',
+    /**
+     * Whether to reuse an existing product instead of creating a new one
+     * @possibleValues `true`, `false`
+     **/
+    'reuseExisting',
+    /**
+     * Whether to the Protecode backend's callback or poll for results
+     * @possibleValues `true`, `false`
+     **/
+    'useCallback',
+    /**
+     * Whether to log verbose information or not
+     * @possibleValues `true`, `false`
+     **/
+    'verbose'
+]
+@Field Set STEP_CONFIG_KEYS = GENERAL_CONFIG_KEYS
```

Comment:
can be removed, see above ...

Reply:
See also my comment above. Adding it to the excludes will solve all related issues.